README.md

```diff
@@ -44,14 +44,15 @@
-| [Mistral-7B](./models/demos/wormhole/mistral7b)                                        | 129th              | 32                   | 9.9 t/s/u - 317 t/s          | 11.0 t/s/u - 352 t/s        | 25             |
-| [Mamba-2.8B](./models/demos/wormhole/mamba)                                            | any                | 32                   | 11.6 t/s/u - 370 t/s         | 16.5 t/s/u - 528 t/s        | 41             |
-| [BERT-Large](./models/demos/metal_BERT_large_11/) (sen/s) [4]                          |                    | 8                    | 270                          | 340                         | 400            |
-| [Stable Diffusion 1.4](./models/demos/wormhole/stable_diffusion) 512x512 (sec/img) [5] |                    | 1                    | 6                            | 5                           | 3              |
-| [ResNet-50](./models/demos/ttnn_resnet) (fps)                                          |                    | 16                   | 4,300                        | 5,550                       | 7,000          |
+| Model                                                                                  | Last Verified Release                                                            | Gen. Token [3]     |  Batch               | End-to-end throughput [1]      | Device throughput [2]        | Target         |
+|----------------------------------------------------------------------------------------|---------------------------------------------------------------------------|--------------------|----------------------|--------------------------------|------------------------------|----------------|
+| [Falcon7B](./models/demos/wormhole/falcon7b)                                           | [v0.51.0-rc13](https://github.com/tenstorrent/tt-metal/tree/v0.51.0-rc13) | 129th              | 32                   | 13.7 t/s/u - 438 t/s          | 19.5 t/s/u - 624 t/s        | 26             |
+| [Mistral-7B](./models/demos/wormhole/mistral7b)                                        | [v0.51.0-rc13](https://github.com/tenstorrent/tt-metal/tree/v0.51.0-rc13) | 129th              | 32                   | 9.90 t/s/u - 317 t/s           | 11.0 t/s/u - 352 t/s         | 25             |
+| [Mamba-2.8B](./models/demos/wormhole/mamba)                                            | [v0.51.0-rc13](https://github.com/tenstorrent/tt-metal/tree/v0.51.0-rc13) | any                | 32                   | 11.6 t/s/u - 371 t/s           | 16.5 t/s/u - 528 t/s         | 41             |
+| [LLaMA-3.1-8B](./models/demos/wormhole/llama31_8b)                                     | [v0.51.0-rc13](https://github.com/tenstorrent/tt-metal/tree/v0.51.0-rc13) | 129th              | 8                    | 8.26 t/s/u - 66.0 t/s     | 9.74 t/s/u - 77.9 t/s   | 14             |
```

Comment:
minor comment, for consistency, use the same decimal points for all t/s/u for all, e.g. 9.7 t/s/u here.